server/src/http/routes/specific.routes/organisme.routes.ts

```diff
@@ -35,7 +41,7 @@ export async function getOrganismeEffectifs(organismeId: ObjectId, anneeScolaire
     "apprenant.adresse.commune",
   ];
 
-  for (const effectifDb of effectifsDb) {
+  for (const effectifDb of effectifs) {
```

Comment:
J'aurai mis `for (const effectif of effectifs)` si on n'utilise plus effectifsDb

Reply:
+1 pour le principe, mais c'est juste qu'il y a une variable scopée à l'intérieur du for qui s'appelle effectif, qui représente +/- un effectif à plat. D'ailleurs, je ne sais pas s'il y a un type pour l'effectif manipulé côté UI.

Reply:
Je vais changer tout ça